pom.xml

```diff
@@ -29,9 +29,8 @@
   </licenses>
 
   <properties>
-    <raml-module-builder.version>35.0.1</raml-module-builder.version>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <java.version>17</java.version>
+    <raml-module-builder.version>35.0.6</raml-module-builder.version>
```

Comment:
Do we still need RMB dependency in pom.xml?

Reply:
Yes, we do need it. It processes ramls/edge-oai-pmh.raml - cfg and produces the following dependency set 
![image](https://github.com/folio-org/edge-oai-pmh/assets/141402754/7689f58a-3e6c-4023-908a-ad761766fb27)